core/src/main/java/com/linecorp/armeria/server/logging/AccessLogger.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright 2017 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.server.logging;
+
+import java.util.List;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.linecorp.armeria.common.logging.RequestLog;
+
+/**
+ * A user may configure an access logger as follows.
+ *
+ * <p>For logback.xml:
+ * <pre>{@code
+ * <configuration>
+ *   <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+ *     <encoder>
+ *       <pattern>%msg%n</pattern>
+ *     </encoder>
+ *   </appender>
+ *
+ *   <logger name="com.linecorp.armeria.logger.access" level="INFO" additivity="false">
+ *     <appender-ref ref="CONSOLE" />
+ *   </logger>
+ *
+ *   <root level="debug">
+ *     <appender-ref ref="STDOUT" />
+ *   </root>
+ * </configuration>
+ * }</pre>
+ *
+ * <p>For log4j.xml:
+ * <pre>{@code
+ * <?xml version="1.0" encoding="UTF-8"?>
+ *   <Configuration status="WARN">
+ *     <Appenders>
+ *       <Console name="Console" target="SYSTEM_OUT">
+ *         <PatternLayout pattern="%msg%n"/>
+ *       </Console>
+ *     </Appenders>
+ *     <Loggers>
+ *       <Logger name="com.linecorp.armeria.logger.access" level="info" additivity="false">
+ *         <AppenderRef ref="Console"/>
+ *       </Logger>
+ *       <Root level="debug">
+ *         <AppenderRef ref="Console"/>
+ *       </Root>
+ *     </Loggers>
+ *   </Configuration>
+ * }</pre>
+ */
+public final class AccessLogger {
+    private static final Logger logger = LoggerFactory.getLogger(AccessLogger.class);
+
+    private static final Logger accessLogger =
+            LoggerFactory.getLogger("com.linecorp.armeria.logger.access");
```

Comment:
How about: `com.linecorp.armeria.accessLog`?

Reply:
As we discussed, I will change it to `com.linecorp.armeria.logging.access`. For consistency, traffic logger's names will be changed as follows:
```
com.linecorp.armeria.logging.traffic.server
com.linecorp.armeria.logging.traffic.client
```